### PR TITLE
WIP: Add date to Mentorship Programme and AMA workshop

### PR DESCRIPTION
**Pull request by MariiaRomanuik**

### CODE REVIEW

**Review by Kislovskiy — APPROVED** [review]
Sounds good, thank you for adding this!

File: data/pages-content/mentorship.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ At the moment we are looking for interested mentors who would like to help our m
 
 ## 1. One-to-One Mentorship Programme
 
-### Open between [Dates - TBA]
+### Open between 15 February - 26 February 2024
```
Comment (thepetk) [inline data/pages-content/mentorship.md:18]: I think we need to wait until the CfP dates are announced because the one-on-one matching depends on them.

**Review by thepetk — CHANGES_REQUESTED** [review]

File: data/pages-content/mentorship.md (hunk 1 above)
Comment (thepetk) [reply data/pages-content/mentorship.md:18]:
```suggestion
### Open until the 27th of February 2024
```

Comment (thepetk) [reply data/pages-content/mentorship.md:18]: @MariiaRomanuik According to https://github.com/EuroPython/website/pull/454 the dates of the CfP are 14 Feb - 6 Mar so a good deadline for the one-to-one is 27th Feb!

**Pull request merged**